src/ResourceManagement/Sql/Sql.Tests/ScenarioTests/Sql2.DataSyncScenarioTest.cs

```diff
@@ -0,0 +1,634 @@
+                    TestUtilities.ValidateOperationResponse(createSyncAgentKeyResponse);
+                    Assert.NotNull(createSyncAgentKeyResponse.SyncAgentKey);
+
+                    //////////////////////////////////////////////////////////////////////
+                    // Get sync agent linked databases
+                    var getSyncAgentLinkedDatabaseResponse = sqlClient.DataSync.ListSyncAgentLinkedDatabase(resGroupName, serverName, syncAgentName1);
+
+                    // Verify the the response from the service contains the right information
+                    TestUtilities.ValidateOperationResponse(getSyncAgentLinkedDatabaseResponse);
+                    Assert.Equal(HttpStatusCode.OK, getSyncAgentLinkedDatabaseResponse.StatusCode);
+
+                    //////////////////////////////////////////////////////////////////////
+                    // Delete sync agent test.
+                    var deleteSyncAgent1 = sqlClient.DataSync.DeleteSyncAgent(resGroupName, serverName, syncAgentName1);
+                    TestUtilities.ValidateOperationResponse(deleteSyncAgent1);
+
+                    sqlClient.Servers.Delete(resGroupName, serverName);
+                }
+                finally
+                {
+                    // Clean up the resource group.
+                    resClient.ResourceGroups.Delete(resGroupName);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Tests for the sync group CRUD operations
+        /// </summary>
+        [Fact]
+        public void SyncGroupCRUDTest()
+        {
+            var handler = new BasicDelegatingHandler();
+
+            using (UndoContext context = UndoContext.Current)
+            {
+                context.Start();
+
+                // Management Clients
+                var sqlClient = Sql2ScenarioHelper.GetSqlClient(handler);
+                var resClient = Sql2ScenarioHelper.GetResourceClient(handler);
+
+                // Variables for server create
+                string serverName = TestUtilities.GenerateName("csm-sql-sgcrud-server");
+                string resGroupName = TestUtilities.GenerateName("csm-rg-sgcrud");
+                string resourceGroupLocation = "West US";
+                string serverLocation = "West US 2";
+                string adminLogin = "testlogin";
+                string adminPass = "Testp@ss";
+                string version = "12.0";
+
+                // Variables for database create
+                string databaseName = TestUtilities.GenerateName("csm-sql-sgcrud-db");
+                string syncDatabaseName = TestUtilities.GenerateName("csm-sql-sgcrud-syncdb");
+                string databaseCollation = "SQL_Latin1_General_CP1_CI_AS";
+                string databaseEdition = "Standard";
+                long databaseMaxSize = 5L * 1024L * 1024L * 1024L; // 5 GB
+                Guid dbSloShared = new Guid("910b4fcb-8a29-4c3e-958f-f7ba794388b2"); // Web / Business
```

Comment:
These are already defined here: https://github.com/Azure/azure-sdk-for-net/blob/master/src/ResourceManagement/Sql/Sql.Tests/Helpers/SqlConstants.cs